setup.py

```diff
@@ -17,56 +17,66 @@
+        return f.read()
 
-setup (
+CHAMELEON_REQUIRES = [
+    'z3c.pt >= 2.1',
+    'z3c.ptcompat >= 1.0',
+
+]
+
+TESTS_REQUIRE = CHAMELEON_REQUIRES + [
+    'zope.testing',
+    'zope.testrunner',
+    'zope.traversing',
+]
+
+setup(
     name='z3c.template',
     version='2.0.1.dev0',
-    author = "Roger Ineichen and the Zope Community",
-    author_email = "zope-dev@zope.org",
-    description = "A package implementing advanced Page Template patterns.",
+    author="Roger Ineichen and the Zope Community",
+    author_email="zope-dev@zope.org",
+    description="A package implementing advanced Page Template patterns.",
     long_description=(
-        read('README.txt')
+        read('README.rst')
         + '\n\n.. contents::\n\n'+
-        read('src', 'z3c', 'template', 'README.txt')
+        read('src', 'z3c', 'template', 'README.rst')
         + '\n' +
-        read('src', 'z3c', 'template', 'zcml.txt')
+        read('src', 'z3c', 'template', 'zcml.rst')
         + '\n' +
-        read('CHANGES.txt')
+        read('CHANGES.rst')
         ),
-    license = "ZPL 2.1",
-    keywords = "zope3 template layout zpt pagetemplate",
-    classifiers = [
+    license="ZPL 2.1",
+    keywords="zope3 template layout zpt pagetemplate",
+    classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Environment :: Web Environment',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
         'Programming Language :: Python',
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope3'],
-    url = 'https://github.com/zopefoundation/z3c.template',
-    packages = find_packages('src'),
-    package_dir = {'':'src'},
-    namespace_packages = ['z3c'],
-    extras_require = dict(
-        test = [
-            'zope.testing',
-            'zope.traversing',
-            ],
-        chameleon = [
-            'z3c.pt >= 2.1',
-            'z3c.ptcompat>=1.0',
-            ],
-        ),
-    install_requires = [
+        'Framework :: Zope3'
+    ],
+    url='https://github.com/zopefoundation/z3c.template',
+    packages=find_packages('src'),
+    package_dir={'': 'src'},
+    namespace_packages=['z3c'],
+    extras_require={
+        'test': TESTS_REQUIRE,
+        'chameleon': CHAMELEON_REQUIRES,
```

Comment:
Maybe it should be called `CHAMELEON_REQUIRE` so it looks more like `TESTS_REQUIRE`.

Reply:
I know that looks awkward, but that's a subject/verb agreement issue in English. Plural subjects, like "tests" (which terminology we inherit from the `setup` keyword) need a plural verb...which happens to be "require" (without an 's'), while singular subjects, like 'chameleon' need a singular verb, which happens to be "requires" (with the 's'). So "tests requires" would be as wrong as "chameleon  require". 

Go figure.

Reply:
@jamadden Okay, you found out what you already knew: I'm not a native speaker.

`CHAMELEONS_REQUIRE` FTW ;)

Reply:
@icemac Honestly, I had to double check your profile to be sure. I really hope I didn't sound condescending. Your English is much better than my German ever was, even when I was actively studying it.

Reply:
@jamadden Thank you. Any help to improve my English is appreciated.